templates/task-question-debug-links.html

```diff
@@ -1,5 +1,5 @@
 {% if DEBUG %}
-<div id="question-debug-links" class="task-question-debug-links">
+<div id="question-debug-links" class="task-question-debug-links" style="border:1px solid black;">
```

Comment:
This css change should go in govready-q.css under the css class `task-question-debug-links`
```suggestion
<div id="question-debug-links" class="task-question-debug-links">
```